proto/federated_service/v1alpha1/federated_service.proto

```diff
@@ -13,11 +13,11 @@ message FederatedService {
   string namespace = 3;
   string cluster_name = 4;
   string trust_domain = 5;
-  string gateway_ip = 6 [deprecated = true];
```

Comment:
Can we leave the dupe here to avoid the breaking change? Or given this was marked as deprecated we shouldn't expect any problems?

Reply:
yeah lets tear it out